.github/workflows/cbmc-update.yml

```diff
@@ -0,0 +1,79 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+name: Attempt CBMC update
+
+on:
+  schedule:
+    - cron: "30 4 * * Mon" # Run this every Monday at 04:30 UTC
+  workflow_dispatch:     # Allow manual dispatching for a custom branch / tag.
+
+permissions:
+  checks: write
+  contents: write
+  issues: write
+  pull-requests: write
+
+jobs:
+  create-cargo-update-pr:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Kani
+        uses: actions/checkout@v3
+
+      - name: Setup Kani Dependencies
+        uses: ./.github/actions/setup
+        with:
+          os: ubuntu-22.04
+
+      - name: Compare CBMC versions
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          grep ^CBMC_VERSION kani-dependencies >> $GITHUB_ENV
+          CBMC_LATEST=$(gh -R diffblue/cbmc release list | grep Latest | awk '{print $1}' | cut -f2 -d-)
+          echo "CBMC_LATEST=$CBMC_LATEST" >> $GITHUB_ENV
+          if [ x$CBMC_LATEST = x$CBMC_VERSION ] ; then
```

Comment:
What's the `x` for?

Reply:
That is to avoid syntax errors should any of those variables evaluate to the empty string. See https://www.sourceware.org/autobook/autobook/autobook_119.html#Test.